### ISSUE TITLE
Simulation working, Google home not

### ISSUE THREAD

**Issue by wouterverduin** [DeanCording/node-red-contrib-google-action#16]
I succesfully integrated this in my NR, Tested with the simulation web app with good results. However: when i talk to my Google Home in the living room i cant invoke a conversation.

Do i need to set something additional for this?

Can i rename the test app to something more friendly like " talk to node red"?

**Comment by Paul-Reed** [comment]
> Can i rename the test app to something more friendly like " talk to node red"?

Yes, from within your Home app, create a custom Routine, and add the command 'use my test app', you can then name the routine whatever you wish.

**Comment by wouterverduin** [comment]
Thanks for your reply; However: I cant seem to get my google Home to reply to custom comments. I tried adding this in the home app with various commands and replys (even not node red related) without succes.

How do i get this with the google home device?

**Comment by jansaris** [comment]
Google doesn't support routines or shortcuts in dutch (yet).
I think we just need to wait a little bit

**Comment by wouterverduin** [comment]
Considering the routines as broken. Is there a way to fix this with the action.json file?

Is it also possible to change the text from "talk to my test app" to something more friendly using that action.json file?
